include/glow/Graph/Graph.h

```diff
@@ -276,6 +276,15 @@ class Function final : public Named {
   FullyConnectedNode *createFullyConnected(llvm::StringRef name,
                                            NodeValue input, size_t outDepth);
 
+  /// Create a row-wise quantized fully connected node. This node is only used
+  /// in quantization. Args \p input and \p B are quantized in regular way, \p W
+  /// is the constant weights and will be row-wise quantized during compile
```

Comment:
maybe specify that it's "during node creation time" since "during compile time" sounds like we have some further optimizations etc.